install/helm/open-match/values.yaml

```diff
@@ -171,6 +171,8 @@ open-match-core:
   pendingReleaseTimeout: 1m
   # Time after a ticket has been assigned before it is automatically delted.
   assignedDeleteTimeout: 10m
+  # Time before a function is considered unavailable after trying to connect to it.
+  functionConnectionTimeout: 300ms
```

Comment:
Client cache is used for more than connecting to a MMF.  Perhaps `rpcConnectionTimeout` with the comment noting MMFs fall under this?

Reply:
Updated